src/global.js

```diff
@@ -119,3 +121,16 @@ export function getDisableSetCookie(): boolean {
 
   return false;
 }
+
+export function getExperimentation(): Experimentation {
+  const experimation = {
+    experience: "",
+    treatment: "",
+  };
+  if (typeof __EXPERIMENTATION__ !== "undefined") {
```

Comment:
I like the idea of using optional chaining inside this block. Do we need this need the undefined check?

Reply:
If you are check `__EXPERIMENTATION__` and it's not defined, you will get undefined error. so we have to add this type check to guaranteed `__EXPERIMENTATION__` is defined.

Reply:
Ah ! thanks for pointing out the webpack global part 😇